asreview/data/base.py

```diff
@@ -240,7 +240,7 @@ def texts(self):
             return self.title
 
         cur_texts = np.array([
-            self.title[i] + " " + self.abstract[i] for i in range(len(self))
+            self.title[i] + self.abstract[i] for i in range(len(self))
```

Comment:
Why concatenate the last word of the title and the first word of the abstract? This affects the models

Reply:
yes you are right. I though this may cause an issue when they were both empty but it doesn't.